### PR TITLE
Close popups on escape

### PR DESCRIPTION
**Pull request by 9inpachi**
Hi,

It'd be cool to close all kind of popups with escape.
I made it possible for some popups using the `EditPanel.vue` as a reference.
I have no way to test it, so you will know better if it's okay.

I think a better way would be to add this in `Popups.vue` for all popups. I am not sure how since I am not familiar with Vue. xD

### CODE REVIEW

**Comment by supertiger1234** [comment]
adding key event to every single popout is gonna be annoying xD I will try adding it to Popups.vue soon.

Thanks a lot though!

File: src/components/app/Popouts/Popouts/AddServer.vue [hunk 1]
```diff
@@ -141,6 +141,10 @@ export default {
         this.closeMenu();
       }
     },
+    keyDownEvent(event) {
+      if (event.keyCode !== 27) return; // 27 = escape
```
Comment (Brecert) [inline src/components/app/Popouts/Popouts/AddServer.vue:145]: `event.keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), changing this to `event.key !== "Escape"` would be better.

Comment (Brecert) [inline src/components/app/Popouts/Popouts/AddServer.vue:145]: the `if isNotKey then stop` pattern is not as understandable as `if IsKey then do { ... }`.
changing this to the second pattern like so would be more readable.
```js
if ( key is pressed) {
  closeMenu
}
```

**Review by brecert — COMMENTED** [review]
I think fishie might do this on their own, but as some hopefully constructive criticism of code I've reviewed this.

**Comment by 9inpachi** [comment]
Thanks for pointing these out @Brecert 
I have this PR opened like an issue so that fishie has this noted.

**Pull request closed without merging**